File: Python/Ex28.py
tecla =str(input("Pressione alguma tecla: "))
n = 1
n2 = 1
for n in range(1,21,1):
      for n2 in range(1,11,1):
        r = n * n2;  
        print(f"{n} * {n2} = {r}")
        n2 = n2 + 1
n = n + 1 
n2 = 1

File: C/Ex28.c
#include <stdio.h>
#include <stdlib.h>
int main(int argc, char *argv[]){
int r, n, n2;
char tecla;
  printf("Pressione alguma tecla: ");
  scanf("%s", &tecla);
for (int n=1; n != 21; n++){
    for (int n2=1; n2 != 11; n2++){
         r = n * n2;  
        printf("%i * %i = %i \n", n, n2, r);
       
    }
}
  return 0;
}